src/components/SideBar.js

```diff
@@ -105,12 +105,21 @@ Menu.propTypes = {
 
 const ProfileMenu = ({ profileImage }) => {
   const { t } = useTranslation();
+  const [, , removeCookie] = useCookiesWithOptions();
 
   const loginMenu = [
     {
       iconName: Icons.SIGN_OUT_ALT,
       children: t('menu.logout'),
-      onClick: () => {},
+      onClick: () => {
+        removeCookie('token');
+        removeCookie('user');
+
+        const queryString = new URLSearchParams({
+          destination: `${window.location.protocol}//${window.location.host}`,
+        }).toString();
+        window.location.href = `${process.env.NEXT_PUBLIC_AUTH_URL}/logout?${queryString}`;
```

Comment:
Just out of curiousity. Could we also use router.push to navigate to an external page?

Reply:
Does seem to work perfectly with router.push. I've changed it